R/data_starry.R

```diff
@@ -15,4 +15,237 @@
+#'   \item{city}{city of the endorser}
+#'   \item{state}{state of the endorser}
+#'   \item{endorser}{name of the endorser}
+#'   \item{endorsee}{name of the endorsee}
+#'   \item{endorser_party}{party of the endorser}
+#'   \item{source}{source link of the endorsement}
+#'   \item{order}{order of the endorsement}
+#'   \item{category}{cateogory of the endorsement}
+#'   \item{body}{body of the endorsement}
+#'   \item{points}{points the endorsement counts for}
+#'   }
+#' @source 2020 endorcement tracker. 
+#' Methodology: 
+#' \url{https://fivethirtyeight.com/methodology/how-our-presidential-endorsement-tracker-works/}.
+"endorsements_2020"
+
+#' How FiveThirtyEight's 2018 Midterm Forecasts Did
+#' 
+#' The raw data behind the story 
+#' 'How FiveThirtyEight's 2018 Midterm Forecasts Did'
+#' \url{https://fivethirtyeight.com/features/how-fivethirtyeights-2018-midterm-forecasts-did/}
+#' 
+#' @format A dataframe with 1518 rows representing forecast results (as of December 3, 2018) and 11 variables:
+#' \describe{
+#'   \item{cycle}{cycle of the election}
+#'   \item{branch}{branch of the election}
+#'   \item{race}{election forecast for the gubernatorial race}
+#'   \item{forecastdate}{the date of the forecast}
+#'   \item{version}{version of the election forecast}
+#'   \item{democrat_winprobability}{the probability of winning for the Democrat}
+#'   \item{republican_winprobability}{the probability of winning for the Republican}
+#'   \item{category}{the predicted political affiliation of the forecast}
+#'   \item{democrat_won}{whether the Democrat won}
+#'   \item{republican_won}{whether the Republican won}
+#'   \item{uncalled}{if a race was uncalled}
+#'   }
+#' @source FiveThirtyEight's 2018 House Forecast \url{https://projects.fivethirtyeight.com/2018-midterm-election-forecast/house/}
+"forecast_results_2018"
+
+#' 2018 Governors Forecast
+#' 
+#' The raw data behind the story 
+#' 'Forecasting the races for governor'
+#' \url{https://projects.fivethirtyeight.com/2018-midterm-election-forecast/governor/}
+#' 
+#' @format A dataframe with 150 rows representing national-level results of the classic, lite, and deluxe gubernatorial forecasts since Oct. 11, 2018. and 11 variables
+#' \describe{
+#' \item{forecastdate}{date of the forecast}
+#' \item{party}{the party of the forecast}
+#' \item{model}{the model of the forecast}
+#' \item{win_probability}{the probability of the corresponding party winning}
+#' \item{mean_seats}{the mean of the number of seats}
+#' \item{median_seats}{the median number of seats}
+#' \item{p10_seats}{the top 10 percentile of number of seats}
+#' \item{p90_seats}{the top 90 percentile of number of seats}
+#' \item{margin}{unknown}
+#' \item{p10_margin}{the margin of p10_seats}
+#' \item{p90_margin}{the margin of p90_seats}
+#' }
+#' @note The original dataset included a meaningless column called "state", and all variables under this column was "US". So this column was removed.
```

Comment:
Since `governor_national_forecast` and `governor_state_forecast` were in the same folder on the fivethirtyeight data repo, could you include a `@seealso` roxygen link to each other? i.e. the help file for `governor_national_forecast` should have a `@seealso` for `governor_state_forecast`, and vice versa?

Reply:
There should be examples of this `@seealso` in help files Chester, Jenn, or I created.

Reply:
yep!